lib/page/house_wikidata.rb

```diff
@@ -12,21 +13,49 @@ def title
       "EveryPolitician: #{country.name} — #{house.name}"
     end
 
-    def country
-      index.country(country_slug)
+    def house_name
+      house.name
```

Comment:
Not quite sure what this is for, but https://github.com/everypolitician/viewer-sinatra/issues/14816 might have solved the underlying problem already.